.goreleaser.yml

```
@@ -0,0 +1,204 @@
+      order: 2
+    - title: 'Code Refactoring'
+      regexp: "^refactor.*"
+      order: 3
+    - title: 'Other Changes'
+      order: 999
+
+release:
+  github:
+    owner: VatsalSy
+    name: CloudPull
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)